spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerPageableQueryOptions.java

```diff
@@ -68,4 +70,16 @@ public SpannerPageableQueryOptions setAllowPartialRead(boolean allowPartialRead)
 		super.setAllowPartialRead(allowPartialRead);
 		return this;
 	}
+
+	@Override
+	public SpannerPageableQueryOptions setTimestamp(Timestamp timestamp) {
```

Comment:
Can we use setTimestampBound instead of setTimestamp and setBoundedTimestamp?
Users could pass TimestampBound.ofMinReadTimestamp(timestamp) or TimestampBound.ofReadTimestamp(timestamp) in that case

Reply:
Done PTAL